recipes/qe/meta.yaml

```diff
@@ -0,0 +1,114 @@
+{% set name = "qe" %}
+{% set version = "6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/QEF/q-e/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: 258b2a8a6280e86dad779e5c56356d8b35dc96d12ff33dabeee914bc03d6d602
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - openmpi-mpifort
+    - make
+  host:
+    - fftw 
+    - fox
+    - libblas
+    - libcblas
+    - libgomp
+    - liblapack
+    - liblapacke
+    - openblas
+    - openmpi
+    - scalapack
+  run:
+    - fftw  
+    - libblas * *openblas
```

Comment:
Should `* *openblas` also be specified in the declaration of `libblas` in the `host`  section?
Looking at the linux-64 job output, there are some warnings thay may indicate it's being built with `netlib` BLAS?

```
WARNING (qe): run-exports library package conda-forge::libcblas-3.8.0-7_h6e990d7_netlib in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (qe): run-exports library package conda-forge::libblas-3.8.0-7_h6e990d7_netlib in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (qe): run-exports library package conda-forge::liblapacke-3.8.0-7_h6e990d7_netlib in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

(just to make sure everything is really ok...)

Reply:
You shouldn't use `openblas` at all in the `run` section. See https://conda-forge.org/docs/maintainer/knowledge_base.html#blas

Reply:
I was following the recommendations in https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=blas#blas

Reply:
@isuruf Now I am confused, because there it states: 

> This would give you a matrix builds for different blas implementations. If you only want to support a specific blas implementation,
> 
> ```
> requirements:
>   host:
>     - openblas
>   run:
>     - libblas * *openblas
>     - openblas
> ```

Reply:
I'm assuming it didn't work with `netlib` packages, and `openblas` was required.. in this case, the recommendation is:

```
requirements:
  host:
    - openblas
  run:
    - libblas * *openblas
    - openblas
```

check if those 4 packages can be removed:

```
    - libblas
    - libcblas
    - liblapack
    - liblapacke
```

Reply:
The issue is that in the build script I specifically link to `openblas`, that is why I require `openblas` at all times.

Reply:
@jan-janssen, that's for advanced use.

> If a package needs a specific implementation’s internal API for more control you can have,

For 99% of packages, you only need,

```
requirements:
  host:
    - libblas
    - libcblas
    - liblapack
    - liblapacke
```

Reply:
I agree, the issue is more that I have trouble to tell fortran to find the right library automatically, that is why I decided to fix it for now. I plan to extend the support to more blas libraries later.

Reply:
I don't understand. You are already using `netlib` libraries in host which is what is used by fortran when building and then when you add `openblas` to `run`, you are restricting the blas libraries for no reason.

Reply:
@isuruf You are right - I was able to remove the openblas statements: 
https://github.com/conda-forge/staged-recipes/pull/11180/commits/5efc8d92a09c5e15ee3d5e04e72c1a0a9a66c639